### PR TITLE
Cc/review

### PR DESCRIPTION
**Pull request by Hermitter**
Initial restructure for MOS documentation.

### CODE REVIEW

File: docs/matrix-core/examples/jstests.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ node -v
 ```
 
 ## Download and Prepare CORE
-```bash
+```language-bash
 git clone https://github.com/matrix-io/matrix-creator-malos;
 cd matrix-creator-malos;
 git submodule update --init;
```
Comment (kdpatino) [inline docs/matrix-core/examples/jstests.md]: It is outdated, check: now malos doesn't have a submodule.

File: docs/matrix-core/getting-started/hello-world.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Download and Prepare
 > You need [NodeJS, and the Node Package Manager (npm)](https://nodejs.org/en/download/) installed on the Raspberry Pi.
-```
+```language-bash
 git clone https://github.com/matrix-io/matrix-creator-malos;
 cd matrix-creator-malos;
 git submodule update --init;
```
Comment (kdpatino) [inline docs/matrix-core/getting-started/hello-world.md]: Outdated, now malos doesn't have a submodules

File: docs/matrix-core/reference/everloop.md [hunk 3]
```diff
@@ -23,11 +23,11 @@ This driver doesn't send any data to a subscribed program.
...
-This section provides an enhanced description of the relevant parts of the [sample source code](https://github.com/matrix-io/matrix-creator-malos/src/js_test/test_everloop.js).
+This section provides an enhanced description of the relevant parts of the <a href="https://github.com/matrix-io/matrix-creator-malos/tree/master/src/js_test" target="_blank">sample source code</a>.
 
 To run the source code of this example you need to execute the following commands:
 
-```
+```language-bash
 git clone https://github.com/matrix-io/matrix-creator-malos.git
 cd matrix-creator-malos
 git submodule update --init
```
Comment (kdpatino) [inline docs/matrix-core/reference/everloop.md]: Again

File: docs/matrix-core/examples/jstests.md (hunk 1 above)
Comment (Hermitter) [reply docs/matrix-core/examples/jstests.md]: removed 
```
 git submodule update --init;
```

File: docs/matrix-core/getting-started/hello-world.md (hunk 2 above)
Comment (Hermitter) [reply docs/matrix-core/getting-started/hello-world.md]: removed 
```
 git submodule update --init;
```

File: docs/matrix-core/reference/everloop.md (hunk 3 above)
Comment (Hermitter) [reply docs/matrix-core/reference/everloop.md]: removed 
```
 git submodule update --init;
```

**Comment by eighteyes** [comment]
Reference page looks a little funky with the padding-top:0. Can we try 0.6em?
image

**Comment by eighteyes** [comment]
for http://127.0.0.1:8000/matrix-os/reference/cli-tool/ where we reference the ping issue, can we make a github issue and cross link it to the docs. so when we clear the issue we remember to update the docs?

**Pull request merged**